nxdrive/gui/folders_dialog.py

```diff
@@ -35,6 +38,11 @@
 DOC_URL = "https://doc.nuxeo.com/n/CBX/#duplicates-behavior"
 
 
+def get_regexp_validator() -> QRegExpValidator:
```

Comment:
Missing docstring. And the "get_" is not really helpful: `regexp_validator()` is enough.